build/build.ps1

```diff
@@ -29,9 +29,23 @@ function Build-One {
 # Fetch TypeScript definitions
 Push-Location (Join-Path $PSScriptRoot ../src/Kernel)
     Try {
-        npm install
+        # npm install writes some output to stderr, which causes PowerShell to throw
+        # unless $ErrorActionPreference is set to 'Continue'.
+        # We also redirect stderr output to stdout to prevent an exception being incorrectly thrown.
+        $OldPreference = $ErrorActionPreference;
+        $ErrorActionPreference = 'Continue'
+        npm install 2>&1 | ForEach-Object { "$_" }
     } Catch {
+        Write-Host $Error[0]
```

Comment:
Would the catch block ever run in this case since we're redirecting all output to stderr?

Reply:
Probably not? I don't know if there's another way for an exception to be thrown here. I thought it wouldn't hurt to keep it there.

Reply:
I guess there's no good way to separate warnings and errors :P